docs/run-local-network.md

```diff
@@ -1,9 +1,9 @@
-# program-simulator
```

Comment:
This is called now nada-run, probably a misunderstanding on our chat, sorry.
https://github.com/NillionNetwork/nillion/pull/1338